dsp/modules/aws_lm.py

```diff
@@ -28,7 +28,6 @@ def __init__(
         max_new_tokens: int,
```

Comment:
Should this file change? We can remove it from the PR I think?

Reply:
Must have gotten mixed up in the rebase. Done.